includes/admin/reporting/reports.php

```diff
@@ -2780,8 +2792,10 @@ function display_export_report() {
 
 				                wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' );
 				                ?>
-                                <input type="hidden" name="edd-export-class" value="EDD_Batch_Customers_Export"/>
-                                <input type="submit" value="<?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?>" class="button-secondary"/>
+								<input type="hidden" name="edd-export-class" value="EDD_Batch_Customers_Export"/>
+								<span>
```

Comment:
I was going to ask why you added the `span` tags here and I still wonder that a little bit, not due to your changes but due to the export forms' behavior. I do see that the rest of the export forms have them, as well as the `<span class="spinner"></span>`, which the customer export does not have. However, I think that the spinner on these forms is never actually implemented, because the [JS appends this entire `notice-wrap` element](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/assets/js/admin/tools/export/index.js#L27) to show a progress bar instead, which I am reasonably certain includes its own spinner.

#7903 is related, and currently milestoned for 3.0.5 (@spencerfinnell what do you think, since you assigned it?), but we may want to consider bumping that up as it impacts all of these boxes, since the `notice-wrap` is no longer the width of the screen.

But all that is to say that maybe we should consider removing the spinners from the export form PHP, as I do not think they're ever exposed. At that point, the `span` around each button may possibly also be dispensed with, but I'm not sure.

Reply:
I added the `<span>` to make it consistent with the other spinner fields and make the styling the same.  With that said, though, I'm not sure that the spinners are being used either.  I thought it was just me, but I couldn't trigger them either.

Reply:
It looks like this will be resolved in #7903